client/src/main/scala/com/dwolla/cloudflare/domain/model/accounts/AccountMember.scala

```diff
@@ -5,4 +5,8 @@ case class AccountMember (
   user: User,
   status: String,
   roles: List[AccountRole]
-)
+) {
+  def uri(accountId: String): String = {
```

Comment:
Moved this to a method on the AccountMember for now. I think we can do better in a future iteration.